src/Extend/Assets.php

```diff
@@ -12,29 +12,37 @@
 namespace Flarum\Extend;
 
 use Flarum\Extension\Extension;
-use Flarum\Frontend\Event\Rendering;
+use Flarum\Frontend\Asset\ExtensionAssets;
+use Flarum\Frontend\FrontendCompilerFactory;
 use Illuminate\Contracts\Container\Container;
-use Illuminate\Events\Dispatcher;
 
 class Assets implements Extender
 {
-    protected $appName;
+    protected $frontend;
 
-    protected $assets = [];
+    protected $css = [];
     protected $js;
 
-    public function __construct($appName)
+    public function __construct($frontend)
     {
-        $this->appName = $appName;
+        $this->frontend = $frontend;
     }
 
-    public function asset($path)
+    public function css($path)
     {
-        $this->assets[] = $path;
+        $this->css[] = $path;
 
         return $this;
     }
 
+    /**
+     * @deprecated
```

Comment:
I'd prefer seeing an indication of why it was deprecated or what to do instead. Referencing an issue or another class method should mostly suffice.

Reply:
Let's just remove this instead? The extenders have not even been released anywhere - we "only" need to update dev-master of our extensions...